src/GitHub.Extensions/StringExtensions.cs

```diff
@@ -36,6 +36,16 @@ public static bool IsNullOrEmpty([AllowNull]this string s)
             return string.IsNullOrEmpty(s);
         }
 
+        public static bool IsNotNullOrEmptyOrWhiteSpace([AllowNull]this string s)
+        {
+            return !s.IsNullOrEmptyOrWhiteSpace();
+        }
+
+        public static bool IsNullOrEmptyOrWhiteSpace([AllowNull] this string s)
+        {
+            return string.IsNullOrEmpty(s) || string.IsNullOrWhiteSpace(s);
```

Comment:
`IsNullOrWhiteSpace` already returns true if the string is empty, so no need to do the extra checks